CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Platform Version 0.9.33 - UNRELEASED
 * Added `DeliverySemantic` to `fluvio-cli`. ([#2508](https://github.com/infinyon/fluvio/pull/2508))
+* CLI: Added ability to delete multiple connectors, smart modules and topics with one command. ([#2518](https://github.com/infinyon/fluvio/pull/2518))
```

Comment:
Could you put the link of the #2427 issue instead?

Reply:
sure thing

Reply:
Just noticed it is a mixed bag of the pull request and issue links. How do you decide what goes into a changelog, issue, or pull request link?

Reply:
We try to reference issues as much of possible. Sometimes, the issue is too broad so we solve the issue with multiple PRs, when we don't have a specific issue for each PR we tend to use the link to the PR.

Reply:
That makes sense! Thanks for the info.